bandwidth-file-spec.txt

```diff
@@ -568,6 +569,32 @@
       The descriptor observed bandwidth mean for this relay in bytes per
       second.
 
+    "vote=" Bool SP
+
+      [Zero or one time.]
+
+      If the value is 0, Tor directory authorities MUST NOT vote on this relay.
```

Comment:
Current Tor implementations do not obey this spec.
Should this line say "SHOULD NOT"?